ElrondNetwork/api.elrond.com
# Trigger metadata refresh on update transactions

## Type

## Problem setting
- NFT metadata update is not showing in the API
  
## Proposed Changes
- Follow update metadata transactions in TP
- Trigger a forceRefresh for metadata

## How to test
- Make a ESDTNFTUpdateAttributes transaction and check if metadata is updated for that NFT
